### ISSUE TITLE
Higher Baud-Rates?

### ISSUE THREAD

**Issue by pbksol** [WLAN-Pi/wconsole#8]
Hi…

Little problem… Regarding the documentation given in the configuration, this…

`#Originally used ports for backwards compatibility adapter #1
2400:telnet:3600:/dev/ttyUSB0:2400 8DATABITS NONE 1STOPBIT banner
4800:telnet:3600:/dev/ttyUSB0:4800 8DATABITS NONE 1STOPBIT banner
9600:telnet:3600:/dev/ttyUSB0:9600 8DATABITS NONE 1STOPBIT banner
19200:telnet:3600:/dev/ttyUSB0:19200 8DATABITS NONE 1STOPBIT banner
8888:telnet:3600:/dev/ttyUSB0:115200 8DATABITS NONE 1STOPBIT banner

#USB-to-serial adapter #1
2401:telnet:3600:/dev/ttyUSB0:2400 8DATABITS NONE 1STOPBIT banner
4801:telnet:3600:/dev/ttyUSB0:4800 8DATABITS NONE 1STOPBIT banner
9601:telnet:3600:/dev/ttyUSB0:9600 8DATABITS NONE 1STOPBIT banner
19201:telnet:3600:/dev/ttyUSB0:19200 8DATABITS NONE 1STOPBIT banner
8889:telnet:3600:/dev/ttyUSB0:115200 8DATABITS NONE 1STOPBIT banner`

should establish a connection using 115200 baud (the 8888 is just a random number to see if it might work at all)… But… Nothing. Telnet tries to connect but stops.

TIO works perfectly on ttyUSB0 using 115200 baud…

Any hints?

Wlanpi 1.9.1

**Comment by jiribrejcha** [comment]
Thanks for reporting.

TIO?

I suspect those ports 8888 and 8889 are not open on the firewall (UFW). You can easily check using the Front Panel Menu System (Menu > Network > UFW ports I believe).

**Comment by pbksol** [comment]
You are right. Ports are closed. But all the newly added default ports 9601, 19201 and so on from the latest version are not listed, too

Why not add all the standard connection speeds to the default configuration? All the routers I deal with use 115k serial speed. Would make it much easier and more convenient.

TIO is a hassle free serial terminal https://github.com/tio/tio

**Comment by jiribrejcha** [comment]
Adding 115200 baud rate listening on ports 11520-11528 (note missing zero in port number please) in the next pull request.
